src/working-key.ts

```diff
@@ -8,16 +8,17 @@ import { WorkingkeyObject } from './types';
  * values or effects to pull / trigger.
  */
 
-const defaultWorkingKey = {
-	// list of custom tags that we've stepped into
-	branch: [],
-	// map of branches to index value (used as a cursor for hooks)
-	branchIndices: {
-		'': 0,
-	},
-} as WorkingkeyObject;
+const defaultWorkingKey = () =>
+	({
+		// list of custom tags that we've stepped into
+		branch: [],
+		// map of branches to index value (used as a cursor for hooks)
+		branchIndices: {
+			'': 0,
+		},
+	} as WorkingkeyObject);
```

Comment:
build a new object (previously we would always point to the same object, which created stale data between tests).